html/pfappserver/root/src/views/Configuration/tenants/_components/TheList.vue

```diff
@@ -0,0 +1,268 @@
+<template>
+  <b-card no-body>
+    <b-card-header>
+      <div class="float-right">
+        <base-input-toggle-advanced-mode
+          v-model="advancedMode"
+          :disabled="isLoading"
+          label-left
+        />
+      </div>
+      <h4 class="mb-0">
+        {{ $t('Tenants') }}
+        <pf-button-help class="ml-1" url="PacketFence_Installation_Guide.html#tenants" />
```

Comment:
The URL should be `PacketFence_Installation_Guide.html#_tenants` (underscore is missing).